runtime/util/vmargs.c

```diff
@@ -626,6 +625,12 @@ optionValueOperations(J9PortLibrary *portLibrary, J9VMInitArgs* j9vm_args, IDATA
 						default:
 							return OPTION_MALFORMED;
 					}
+
+					if (action==GET_MEM_VALUE) {
+						oldValue = value;
+						value = (value +  sizeof(UDATA) - 1) & ~(sizeof(UDATA) - 1);		/* round to nearest pointer value */
+						if (value < oldValue) return OPTION_OVERFLOW;
```

Comment:
I don't believe the coding standards allow this. Pls add `{}`.
https://github.com/eclipse-omr/omr/blob/master/doc/CodingStandard.md